autodoc/admin-api/data/admin-api.lua

```diff
@@ -1464,6 +1463,26 @@ return {
             HTTP 204 No Content
             ```
           ]]
+        },
+        PATCH = {
+          title = [[Update Target]],
+          description = [[
+            Update a target.
+          ]],
+          endpoint = [[
+            <div class="endpoint patch indent">/upstreams/{upstream name or id}/targets/{host:port or id}</div>
+
+            {:.indent}
+            Attributes | Description
+            ---:| ---
+            `upstream name or id`<br>**required** | The unique identifier **or** the name of the upstream for which to update the target.
+            `host:port or id`<br>**required** | The host:port combination element of the target to update, or the `id` of an existing target entry.
+          ]],
+          response = [[
+            ```
+            HTTP 201 Created
```

Comment:
Isn't this `200 OK` instead @locao?

Reply:
Actually it's `204 No Content` now, but we are supposed to fix this to use the same standard as other end-points. Sorry, @javierguerragiraldez.